docs/subnets/customize-a-subnet.md

```diff
@@ -376,7 +404,23 @@ interface NativeMinterInterface {
 }
 ```
 
-_Note: Both `ContractDeployerAllowList` and `ContractNativeMinter` can be used together._
+#### Trustless Mode
+
+It's possible to enable this precompile without an admin address. In this mode you can define a list of addresses that will receive native coins. This mode is useful for networks that requires one-time mint only without a trusted party. To enable this mode, you need to specify a map of addresses with their corresponding mint amounts in `initialMint` field in your genesis or upgrade file:
+
+```json
+{
+  "contractNativeMinterConfig": {
+    "blockTimestamp": 0,
+    "initialMint": {
+      "0x8db97C7cEcE249c2b98bDC0226Cc4C2A57BF52FC": "1000000000000000000",
+      "0x10037Fb06Ec4aB8c870a92AE3f00cD58e5D484b3": "0xde0b6b3a7640000"
+    }
+  }
+}
+```
+
+In the amount field you can specify either decimal or hex string. This will mint 1000000000000000000 (equivalent of 1 Coin) to both addresses. Note that these are both in string format. "0xde0b6b3a7640000" hex is equivalent to 1000000000000000000. For further information about trustless precompiles see [Trustless Precompile Activation](#trustless-precompile-activation).
```

Comment:
```suggestion
In the amount field you can specify either decimal or hex string. This will mint 1000000000000000000 (equivalent of 1 Native Coin denominated as 10^18) to both addresses. Note that these are both in string format. "0xde0b6b3a7640000" hex is equivalent to 1000000000000000000. For further information about trustless precompiles see [Trustless Precompile Activation](#trustless-precompile-activation).
```